references/plugins/references.go

```diff
@@ -510,9 +631,12 @@ func (ref *ParseReference) getHELMPrintableDefaultValue(dataType string, value i
 	return defaultValueMap[dataType]
 }
 
-// generateSpecification generates Specification part for reference docs
-func (ref *MarkdownReference) generateSpecification(capabilityName string) string {
-	return fmt.Sprintf("```yaml%s```", ConfigurationYamlSample[capabilityName])
+// generateSample generates Specification part for reference docs
+func (ref *MarkdownReference) generateSample(capabilityName string) string {
+	if _, ok := ConfigurationYamlSample[capabilityName]; ok {
```

Comment:
```suggestion
       // TODO(zzxwill): we should generate the sample automatically instead of maintain hardcode example.
	if _, ok := ConfigurationYamlSample[capabilityName]; ok {
```